internal/metrics/waste.go

```diff
@@ -31,16 +31,20 @@ import (
 )
 
 const (
-	metricCacheCleanupInterval = 6 * time.Hour
+	demandFulfilledAgeCleanUp = 6 * time.Hour
 )
 
 var (
 	beforeDemandCreation = tagInfo{
 		tag:              metrics.MustNewTag(schedulingWasteTypeTagName, "before-demand-creation"),
 		slowLogThreshold: 1 * time.Minute,
 	}
-	afterDemandFulfilled = tagInfo{
-		tag:              metrics.MustNewTag(schedulingWasteTypeTagName, "after-demand-fulfilled"),
+	afterDemandFulfilledNoFailures = tagInfo{
```

Comment:
I think we should keep `afterDemandFulfilled` and always mark it whenever we have a non zero demand fulfilled time, so we can compare